tests/chainer_tests/test_optimizer.py

```diff
@@ -638,79 +629,63 @@ def setUp(self):
             lr = self.loss_scale
             for i in range(2):
                 self.target[i].param._loss_scale = self.loss_scale
```

Comment:
Doesn't have to be addressed in this PR but there must be a better way to test the loss scale other than modifying a private attribute 🤔

Reply:
Possibly. Indeed it's something better discussed at another place.